app/Classes/MailClass.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Classes;
+
+use Mail;
+
+class MailClass
```

Comment:
I don't really like that we name this as `Classes`, but okay. Since we don't have any ideas, lets leave it as it is. But please think about it.